test/unit/notifications.server.test.ts

```diff
@@ -0,0 +1,33 @@
+import sendNotification from '../../app/lib/notifications.server';
+const { NOTIFICATIONS_EMAIL_USER } = process.env;
+describe('Notification server lib function test', () => {
+  const recipient: string = 'dev@starchart.invalid';
+  const subject: string = 'Sample subject';
+  const text: string = 'Sample text';
+
+  test('sending email notification using sendNotification() ', async () => {
+    const response = await sendNotification(recipient, subject, text);
+    expect(response).toEqual(
+      expect.objectContaining({
+        accepted: expect.arrayContaining([recipient]),
+      })
+    );
+  });
+
+  test('verifying MailHog received test email notification via SendNotification() ', async () => {
+    const response = await fetch(`http://localhost:8025/api/v2/search?kind=to&query=${recipient}`);
+    const responseToJSON = await response.json();
+    const mostRecentMessage = responseToJSON.items[0];
```

Comment:
This is still problematic.  We should test against `.items` vs. assuming anything about the order or position in the array.  Also, this test now depends on the previous test being run before it, which you can't assume.  There should be no order assumed between test runs.